src/test/java/org/folio/auth/authtokenmodule/AuthTokenTest.java

```diff
@@ -1023,4 +1043,161 @@ private String getMagicPermission(String endpoint) {
     return String.format("%s.execute", Base64.encode(endpoint));
   }
 
+  @Test
+  public void testStoreSaveRefreshToken(TestContext context) {
+    var ts = new RefreshTokenStore(vertx, tenant);
+    var rt = new RefreshToken(tenant, "jones", userUUID, "http://localhost:" + port);
+    ts.saveToken(rt)
+      .compose(x -> ts.checkTokenNotRevoked(rt))
+      .onComplete(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void testStoreRefreshTokenNotFound(TestContext context) {
+    var ts = new RefreshTokenStore(vertx, tenant);
+    // A RefreshToken which doesn't exist is treated as revoked.
```

Comment:
This comment should also been added to the checkTokenNotRevoked method.

Reply:
Done